### DISCUSSION TITLE
Website is crashing iphone

### DISCUSSION THREAD

**Discussion by BURROO** [gatsbyjs/gatsby#28219] [Help]
I have no idea what I did there; It is a website I am working on, that worked fine on mobile devices a while ago.
Now it is crashing the iphone (any version, any browser i was able to test), I didn't get the chance to check it on Android yet.
If literally crashes the browser and makes the phone reboot.
For hours I am trying to find the solution anywhere possible. But at this point I have no more clues. 
The website is built with gatsby and hosted on netlify. I restuctered large parts of the code and tried to replace certain plugin with alternatives… I am still not sure if this issues has anything to do with gatsby. I also found reports of certain html/css combinations that were crashing IOS devices, but that didn't help me much either.
I appreciate any help or hints.

This is what my package.json looks like:

"dependencies": {
    "dotenv": "^8.2.0",
    "gatsby": "^2.27.0",
    "gatsby-cli": "^2.12.107",
    "gatsby-image": "^2.4.21",
    "gatsby-plugin-manifest": "^2.4.34",
    "gatsby-plugin-offline": "^3.2.31",
    "gatsby-plugin-preact": "^4.2.0",
    "gatsby-plugin-purgecss": "^5.0.0",
    "gatsby-plugin-react-helmet": "^3.3.14",
    "gatsby-plugin-sass": "^2.3.16",
    "gatsby-plugin-sharp": "^2.6.40",
    "gatsby-plugin-transition-link": "^1.20.2",
    "gatsby-source-filesystem": "^2.3.34",
    "gatsby-source-graphql": "^2.7.6",
    "gatsby-transformer-sharp": "^2.5.17",
    "lazysizes": "^5.2.2",
    "preact": "^10.5.7",
    "preact-render-to-string": "^5.1.12",
    "prop-types": "^15.7.2",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-helmet": "^5.2.1",
    "react-id-swiper": "^2.4.0",
    "react-infinite-scroller": "^1.2.4",
    "react-pose": "^4.0.10",
    "slick-carousel": "^1.8.1",
    "swiper": "^6.3.4"
  },
  "devDependencies": {
    "gatsby-plugin-webpack-bundle-analyser-v2": "^1.1.13",
    "node-sass": "^4.14.1",
    "prettier": "^1.19.1"
  },

And this is the link to the website:
WARNING IF OPENING ON THE PHONE: It will crash and reboot your iphone:
https://objective-wright-1a0dab.netlify.app/

**Comment by tonicaktas** [comment]
Hi
Did u solve this one? I am having similar issue and it drives me crazy. Site crashes on ios devices, but not all. I cannot debug since the site gets blank.

**Reply by BURROO** [reply]
In my case I removed lazysizes and infinite scroll from the projects and I did I bunch of other optimization on the website. As there were many small images loaded on the Index Page, in combination with a heavy use of javascript (which was required by the design), that might have caused the phone to crash. However, unfortunately I am not able to pin it down to one solution.

**Reply by tonicaktas** [reply]
Thank u for the reply. I have same setup when it comes to design. Lots of small images in index and lots of components in index. I don't have lazysizes and infinte scroll. 
U can check the site on https://www.restaurangerboras.se....
